.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+jobs:
+  bump-version:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
Updating to use checkout@v4. We should apply this on our other workflows, but that can be a future update